.gitignore

```diff
@@ -12,3 +12,17 @@ target
 #**/*.rs.bk
 certs/
 config/
+
```

Comment:
This is also no longer required as we are moving android & java stuff to a different repository